askama_derive/src/input.rs

```diff
@@ -356,7 +356,8 @@ fn extension(path: &Path) -> Option<&str> {
 #[derive(Debug)]
 pub(crate) enum Source {
     Path(String),
-    Source(String),
+    /// Contains the text of the template and its path (if it's not an inline template).
+    Source(String, Option<PathBuf>),
```

Comment:
This is confusing. If my memory serves correctly, `Source::Source` exactly describes "inline templates", so the `Option<PathBuf>` should actually be on the `Path` variant? Did you test this end to end?

Reply:
Let me double-check. Maybe I got confused on the way...

Reply:
I can't reproduce what I did yesterday so I reverted this change.